### PR TITLE
Update/home browser tests

### PR DESCRIPTION
**Pull request by Dremars**
Added browser tests for home view and move PaperButton attribute testing to jest

### CODE REVIEW

**Comment by Dremars** [comment]
Wait for #845 to be merged and move getLocation to utility before merge

File: browserTests/views/homeTest.js [hunk 1]
```diff
@@ -1,17 +1,74 @@
...
+
 fixture`Home view test`
-  .page`http://${server.address}:${server.port}/fi/`
+  .page`${viewUrl}`
   .beforeEach(async () => {
     await waitForReact();
   });
 
-test('PaperButtons work correctly', async (t) => {
+test.only('PaperButtons work correctly', async (t) => {
```
Comment (mikkoisa) [inline browserTests/views/homeTest.js:19]: Forgot only-parameter here

**Review by mikkoisa — CHANGES_REQUESTED** [review]
See comment. Tests look good

File: browserTests/views/homeTest.js (hunk 1 above)
Comment (Dremars) [reply browserTests/views/homeTest.js]: Fixed in 3ed9f3a

**Review by mikkoisa — APPROVED** [review]

**Pull request merged**